### ISSUE TITLE
Accessibility score to be improved.

### ISSUE THREAD

**Issue by rmathew1011** [TAMULib/weaver-components#321]
At present the score for accessibility has been brought down to 0.85 - the min score assertion should be maintained at 0.9
